README.md

```diff
@@ -1,3 +1,6 @@
 # workspace
 
 Themes, utilities and settings to optimize our development workflow
+
+#Setup
+Run ./setup/setup.sh for installing a new Mac instance.
```

Comment:
This might not seem obvious enough that setup is the goto runner for every setup script. Could we rename it to run.sh or something? Then you could do `run.sh zsh`

Reply:
Keeping shell as discussed.